Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -156,10 +163,16 @@ private static bool CanBeConfigured(MethodInfo method)
             if (methodCallParams.Any(exp => exp == null))
                 return null;
 
-            //e.g. "x.Method(It.IsAny<string>(), out parameter)"
-            var methodCall = Expression.Call(lambdaParam, method, methodCallParams);
+            Expression methodCall;
+            if (DelegateSpecification.IsSatisfiedBy(mockedType))
+                // e.g. "x(It.IsAny<string>(), out parameter)"
+                methodCall = Expression.Invoke(lambdaParam, methodCallParams);
+            else
+                //e.g. "x.Method(It.IsAny<string>(), out parameter)"
```

Comment:
nit: inconsistent spacing. Elsewhere is "// e.g."

Reply:
A good catch 👍 Fixed this and a few more similar places around.